test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -1569,20 +1572,18 @@ context('action sidebar', function() {
     cy
       .routesForPatientAction()
       .routeAction(fx => {
-        fx.data.id = '12345';
-        fx.data.attributes.name = 'Program Action Name';
-        fx.data.attributes.outreach = 'patient';
-        fx.data.attributes.sharing = 'responded';
-        fx.data.relationships['program-action'] = { data: { id: '1' } };
-        fx.data.relationships.form = { data: { id: '11111' } };
-        fx.data.relationships.state = { data: { id: '55555' } };
-        fx.included.push({
-          id: '1',
-          type: 'programs',
+        fx.data = getAction({
+          id: '12345',
```

Comment:
would love to remove the non uuid ids from the `getFoo`s in the tests

Reply:
I removed several more. Think I have all of these removed, besides the ones we need to keep.